docs/README.md

```diff
@@ -460,6 +460,24 @@ Supported events:
 'pull_request.*', 'pull_request_review.*'
 
 ```
+
+### request_review
+Creates comments in issues and/or pull requests depending on the event specified in the `when` tag.
+
+```yml
+- do: request_review
+  reviewers: ['name1', 'name2']
+```
+An API call will only be made if the reviewers are not requested already.
```

Comment:
I would change it to  "This is only enforced if the reviewers who are not already requested."

Reply:
addressed had to change the wording a bit